core/lib/via_btc_client/src/indexer/mod.rs

```diff
@@ -318,16 +318,26 @@ impl BitcoinInscriptionIndexer {
                 // }
             }
             FullInscriptionMessage::ValidatorAttestation(va) => {
-                debug!("Processing ValidatorAttestation message");
-                if state.proposed_sequencer.is_some() {
-                    // warn!("sender_address : {:?}", sender_address);
-                    if let Some(proposed_txid) = state.proposed_sequencer_txid {
-                        if va.input.reference_txid == proposed_txid {
-                            state
-                                .sequencer_votes
-                                .insert(state.verifier_addresses[0].clone(), va.input.attestation);
+                if let Some(sender_address) = parser::get_btc_address(&va.common, network) {
```

Comment:
so let's remove this check since `get_btc_address` is wrong

Reply:
Actually I wanted to check here if `sender_address` exists (even if it is not calculated correctly by the `get_btc_address`), because we need to populate the `sequencer_votes` map (line 332) otherwise the bootstrap process won't  be completed.